need_imu: false
need_output_data: true
use_attitude_safety_catch: true
max_attitude_angle: 0.43
takeoff_height: 0.0
